### ISSUE TITLE
x64 VSCode for Windows

### ISSUE THREAD

**Issue by rongjiecomputer** [Microsoft/vscode#27010]
- VSCode Version: 1.12.2
- OS Version: Windows 10 x64

Can we have 64-bits version of VSCode?

Reasons:

- Chrome and Node.js depend on V8 which has a JIT compiler. 32-bits version of V8 can only generate ia32 native code. 64-bits version of V8 can generate faster SIMD native code and not all of it is available in 32-bits (like AVX1 and AVX2). Since VSCode spends quite a large portion of time running JS code, there will a performance win to use x64 JIT compiler.
- In order to improve stability, performance, and security, Chrome automatically migrates to 64-bits Chrome for 64-bits OS users with more than 4GB RAM but use 32-bits Chrome. VSCode is unlikely to be attacked, but who can refuse more security and stability? [Reference](https://chromereleases.googleblog.com/2017/05/stable-channel-update-for-desktop_9.html)
- [Atom editor](https://atom.io/) provides both 32-bits and 64-bits executable for Windows.

Experiment:

- Download [prebuilt electron-v1.7.0-win32-x64](https://github.com/electron/electron/releases/download/v1.6.10/electron-v1.6.10-win32-x64.zip) and place it under `C:\electron`.
- Copy `C:\Program Files (x86)\Microsoft VS Code\resources\app` to `C:\electron\resources\app`
 - Launch `C:\electron\electron.exe`.

It did launch the outer UI of VSCode, it even picked up the editor theme of my original VSCode, but no content was shown.

I opened the developer tool and found the bug: some node.js modules are native addons built as 32-bits    .node (.dll) which is not compatible with 64-bits program. This can be configured in `node-gyp` to build 64-bits binaries.

What is the main concern not doing this?

**Comment by yfdyh000** [comment]
Duplicate of #507.

**Comment by egamma** [comment]
Closing as duplicate and it is already on the plan, pls https://github.com/Microsoft/vscode/issues/26068
